app/javascript/components/close_contact/CloseContact.js

```diff
@@ -120,8 +120,10 @@ class CloseContact extends React.Component {
                   className="form-square"
                   value={this.state.notes || ''}
                   placeholder={this.closeContactNotePlaceholder}
+                  maxLength="2000"
                   onChange={this.handleChange}
                 />
+                <Form.Label className="close-contact-character-limit"> {2000 - this.state.notes.length} characters remaining </Form.Label>
```

Comment:
Would add a check just in case `this.state.notes` is `null` or `undefined` here before getting `.length` on it.

Reply:
I thought the same thing, and initially had a `?` operator in there. Then I noticed that it is initialized as `notes: this.state.notes || ''`. If `this.state.notes` is null, then it will be an empty string (which has `.length`). If not, it will be a string (which also has `.length`)